R/process_tests_sc_sandpit.R

```diff
@@ -0,0 +1,144 @@
+      dplyr::filter(!(is.na(.data$chi_upi))) %>%
+      dplyr::group_by(.data$social_care_id, .data$sending_location) %>%
+      dplyr::distinct(.data$chi_upi, .keep_all = TRUE) %>%
+      dplyr::mutate(distinct_chi_count = dplyr::n_distinct(.data$chi_upi)) %>%
+      dplyr::filter(distinct_chi_count > 1) %>%
+      dplyr::distinct(.data$social_care_id, .data$sending_location, .keep_all = TRUE) %>%
+      dplyr::mutate(sc_id_multi_chi = 1) %>%
+      create_sending_location_test_flags(.data$sending_location) %>%
+      dplyr::ungroup() %>%
+      dplyr::rename(
+        sc_id_multi_chi_Aberdeen_City = Aberdeen_City,
+        sc_id_multi_chi_Aberdeenshire = Aberdeenshire,
+        sc_id_multi_chi_Angus = Angus,
+        sc_id_multi_chi_Argyll_and_Bute = Argyll_and_Bute,
+        sc_id_multi_chi_City_of_Edinburgh = City_of_Edinburgh,
+        sc_id_multi_chi_Clackmannanshire = Clackmannanshire,
+        sc_id_multi_chi_Dumfries_and_Galloway = Dumfries_and_Galloway,
+        sc_id_multi_chi_Dundee_City = Dundee_City,
+        sc_id_multi_chi_East_Ayrshire = East_Ayrshire,
+        sc_id_multi_chi_East_Dunbartonshire = East_Dunbartonshire,
+        sc_id_multi_chi_East_Lothian = East_Lothian,
+        sc_id_multi_chi_East_Renfrewshire = East_Renfrewshire,
+        sc_id_multi_chi_Falkirk = Falkirk,
+        sc_id_multi_chi_Fife = Fife,
+        sc_id_multi_chi_Glasgow_City = Glasgow_City,
+        sc_id_multi_chi_Highland = Highland,
+        sc_id_multi_chi_Inverclyde = Inverclyde,
+        sc_id_multi_chi_Midlothian = Midlothian,
+        sc_id_multi_chi_Moray = Moray,
+        sc_id_multi_chi_Na_h_Eileanan_Siar = Na_h_Eileanan_Siar,
+        sc_id_multi_chi_North_Ayrshire = North_Ayrshire,
+        sc_id_multi_chi_North_Lanarkshire = North_Lanarkshire,
+        sc_id_multi_chi_Orkney_Islands = Orkney_Islands,
+        sc_id_multi_chi_Perth_and_Kinross = Perth_and_Kinross,
+        sc_id_multi_chi_Renfrewshire = Renfrewshire,
+        sc_id_multi_chi_Scottish_Borders = Scottish_Borders,
+        sc_id_multi_chi_Shetland_Islands = Shetland_Islands,
+        sc_id_multi_chi_South_Ayrshire = South_Ayrshire,
+        sc_id_multi_chi_South_Lanarkshire = South_Lanarkshire,
+        sc_id_multi_chi_Stirling = Stirling,
+        sc_id_multi_chi_West_Dunbartonshire = West_Dunbartonshire,
+        sc_id_multi_chi_West_Lothian = West_Lothian
+      ) %>%
+      dplyr::select(.data$sc_id_multi_chi, .data$sc_id_multi_chi_Aberdeen_City:.data$sc_id_multi_chi_West_Lothian) %>%
+      calculate_measures(measure = "sum")
+
+    output <- list(
+      missing_tests,
+      latest_flag_tests,
+      sc_id_multi_chi
+    ) %>%
+      purrr::reduce(dplyr::full_join, by = c("measure", "value"))
+
+    return(output)
+  } else if (type == "client" | type == "at" | type == "ch" |
+    type == "hc" | type == "sds") {
+    output <- data %>%
+      # create test flags
+      dplyr::mutate(
+        unique_scid = dplyr::lag(.data$social_care_id) != .data$social_care_id,
```

Comment:
```suggestion
        unique_sc_id = dplyr::lag(.data$social_care_id) != .data$social_care_id,
```
Maybe use `unique_sc_id` for consistency.